styles.css

```diff
@@ -30,10 +23,11 @@ nav {
 
 @media (min-width: 375px) and (max-width: 768px) {
```

Comment:
- [ ] Note that you are expected to build your portfolio mobile-first, this means that all mobile stylings should be your default styling (outside of media queries), and then desktop stylings will be in a media query. Kindly remove all mobile stylings from the media query.